helm/sematic-server/values.yaml

```diff
@@ -5,6 +5,8 @@ auth:
 aws:
   enabled: true
   storage_bucket: sematic-dev
+app_config:
+  worker_kubernetes_sa: default
```

Comment:
the following key hierarchy would be more extensible in the future:

```
worker:
  service_account:
    create: True
    name: default
```

Reply:
changed